packages/core-ethereum/src/index.ts

```diff
@@ -90,8 +90,8 @@ export default class HoprEthereum {
     return this.indexer.getPublicKeyOf(addr)
   }
 
-  public getRandomChannel() {
-    return this.indexer.getRandomChannel()
+  public getRandomOpenRoutingChannel() {
```

Comment:
there is no such thing as a routing channel any more

Reply:
after #1857 is merged right?

Reply:
exactly